helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -78,8 +78,8 @@
    */
   @Deprecated
   public BestPossibleExternalViewVerifier(String zkAddr, String clusterName, Set<String> resources,
-      Map<String, Map<String, String>> errStates, Set<String> expectLiveInstances) {
-    super(zkAddr, clusterName);
+      Map<String, Map<String, String>> errStates, Set<String> expectLiveInstances, int waitTillVerify) {
```

Comment:
Note that these constructors are deprecated and suggest to use builder. So you may also want to add this new parameter to the builder.

Reply:
This is good point.

In fact, we already added them in the builder. See this pull, https://github.com/apache/helix/pull/1328/files

The thing is that in  pull 1328, we missed some places which is addressed in this pull.

The big pull #1227 validated that all missing ones are all addressed here. No more missing ones.